### PR TITLE
issue #985

### PR DESCRIPTION
**Pull request by Mariatta**
Attempt to address issue #985

removed braile, embossed, tty value from media attribute since they've been deprecated
replaced inline table attributes: cellpadding, cellspacing, width, border, used css
adjust values for role attribute
These two errors remained, but I'm not familiar with these. Any suggestion of what to do here?
Bad value cleartype for attribute http-equiv on element meta.
Bad value imagetoolbar for attribute http-equiv on element meta.

### CODE REVIEW

File: templates/components/success-story.html [hunk 1]
```diff
@@ -13,15 +13,15 @@ <h2 class="widget-title"><span aria-hidden="true" class="icon-success-stories"><
                                 <a href="{{ story.get_absolute_url }}">{{ story.pull_quote }}</a>
                             </blockquote>
 
-                            <table cellpadding="0" cellspacing="0" border="0" width="100%" class="quote-from">
+                            <table class="quote-from">
```
Comment (berkerpeksag) [inline templates/components/success-story.html:16]: Thanks for cleaning up this!

File: templates/components/success-story.html [hunk 2]
```diff
@@ -13,15 +13,15 @@ <h2 class="widget-title"><span aria-hidden="true" class="icon-success-stories"><
...
                                 <tbody>
                                     <tr>
                                         {% comment %}
                                         This image tag should be for a Company logo, if they have supplied one
                                         Currently this path is for a static logo sample, which no longer exists
                                         <td><a href="{{ story.get_absolute_url }}"><img src="{{ STATIC_URL }}img/ILM_logo.png"></a></td>
                                         {% endcomment %}
-                                        <td><p><a href="{{ story.get_absolute_url }}">{{ story.name }}</a> <em>by {{ story.author }}</em></p></td>
+                                        <td><p><a href="{{ story.get_absolute_url }}">aaa{{ story.name }}</a> <em>by {{ story.author }}</em></p></td>
```
Comment (berkerpeksag) [inline templates/components/success-story.html]: This looks like a leftover from a debugging session.

Comment (Mariatta) [inline templates/components/success-story.html]: Oops.. thanks for catching. I'll make another commit right away. Sorry about this.

**Comment by berkerpeksag** [comment]
Looks good to me, thanks! I still need to test this in my local setup.

> Bad value cleartype for attribute http-equiv on element meta.

This is probably harmless and can stay.

> Bad value imagetoolbar for attribute http-equiv on element meta.

This is for IE's image toolbar and we can probably remove it. It would be nice to check if it's still useful in Edge.

**Comment by Mariatta** [comment]
Hi, just wanted to follow up on this pull request. Wondering if I should make additional adjustments?
Thanks.

**Comment by berkerpeksag** [comment]
Sorry, I'm pretty busy with Python 3.6.0 b1 this week (deadline is on 12th) I will try to get this in next week, thanks! In the meantime, it would be great if you can remove the "imagetoolbar" tag I mentioned in https://github.com/python/pythondotorg/pull/992#issuecomment-241179011.

**Comment by Mariatta** [comment]
No problem. I will get the imagetoolbar tag removed later today.
Thanks.

**Comment by Mariatta** [comment]
Let me know anything else that I should improve on this.
Is there a process in place for testing against different browsers?
Thank you.

**Comment by berkerpeksag** [comment]
@Mariatta do you have time to rebase the validate-html branch? (and apologies for my very late response!)

**Comment by Mariatta** [comment]
😅 I also forgot about this PR. If you're in no rush, I'll be able to get back to this in a couple more weeks.

**Comment by berkerpeksag** [comment]
Of course! I can also work on it, I still have a few hours left from my grant :)

**Comment by Mariatta** [comment]
@berkerpeksag Sounds awesome! I really don't mind if you can take this over. 
I'm on a new laptop and will take more time for me to get my local environment setup again.

**Comment by JacobCoffee** [comment]
Closing in favor of #2536 since I can't push to forks.

Commit credits will remain.

**Pull request closed without merging**